sw/device/silicon_creator/rom/data/rom_testplan.hjson

```diff
@@ -134,400 +121,1008 @@
 
-            - Load new firmware of version N.
-            - Attempt to upgrade to version N+1.
+            - Verify that
+              - `CREATOR_SW_CFG_FLASH_DATA_DEFAULT_CFG` controls the default scrambling, ecc, and he
+                settings for the data partitions.
+              - `CREATOR_SW_CFG_FLASH_INFO_BOOT_DATA_CFG` controls the scrambling, ecc, and he
+                settings for the boot partitions `kFlashCtrlInfoPageBootData0` and
+                `kFlashCtrlInfoPageBootData1`.
+            - Verify that `kFlashCtrlInfoPageCreatorSecret` is locked down.
+            - Verify that flash_ctrl is initialized.
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       milestone: V2
       tests: []
     }
+
     {
-      name: rom_e2e_upgrade_min_sec_version
-      desc: '''Verify that ROM checks minimum security version before booting.
+      name: rom_e2e_keymgr_init
+      desc: '''Verify that ROM initializes the keymgr correctly.
 
-            - Load ROM_EXT that updates min_security_version past its own version.
-            - Attempt to boot again.
-            - Verify that the same ROM_EXT cannot be booted again.
+            - Verify that keymgr is at the "reset" state.
+            - Verify that
+              - attestation sw binding equals
+                - the digest of the ROM_EXT if `OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN` is
+                  `kHardenedBoolTrue`, and
+                - the `binding_value` field of the manifest, otherwise,
+              - sealing sw binding equals the `binding_value` field of the manifest, and
+              - max creator key version equals the `max_key_version` field of the manifest,
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       milestone: V2
       tests: []
     }
+
     {
-      name: rom_e2e_upgrade_power_loss
-      desc: '''Verify that losing power during a ROM_EXT upgrade does not brick the chip.
+      name: rom_e2e_static_critical
+      desc: '''Verify that ROM initializes the data structures in the `.static_critical` section
+            properly.
 
-            - Initiate a ROM_EXT update.
-            - After the new ROM_EXT is loaded into flash, trigger a reset.
-            - Verify that the chip can boot.
+            - Verify that `boot_measurements` holds the digest of the next stage.
+              - Don't specify any usage constraints to be able to compute the actual hash in the second stage.
+            - Verify that `sec_mmio_ctx` is valid using `sec_mmio_check_values()` and
+              `sec_mmio_check_counters()`.
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       milestone: V2
       tests: []
     }
+
     {
-      name: rom_rv_dm_perform_debug
-      desc: '''Verify that Ibex enters debug mode when triggered through JTAG
+      name: rom_functests
+      desc: '''Determine which functests can be executed using ROM.
```

Comment:
should we add a similar point for chip-level tests?

Reply:
It would probably increase our confidence but I'm not sure about the how long they will take. We can evaluate after compiling this list. I'll resolve this comment after adding this comment to the corresponding issue.

Reply:
Added to #14503